Libraries/Text/RCTTextField.m

```diff
@@ -279,4 +296,27 @@ - (BOOL)resignFirstResponder
   return result;
 }
 
+
++(NSMutableDictionary*)getAcessoryMapping {
+  static dispatch_once_t onceToken = 0;
+  
+  dispatch_once(&onceToken, ^{
+    textFieldAccessoryMapping = [[NSMutableDictionary alloc] init];
+  });
+  
+  return textFieldAccessoryMapping;
+}
+
+/**
+ * The method to allow application register handle to produce inputAccessoryView for particular keyboard type
+ * handler presents instancdispatch_oncee of KeyboardAccessory type
+ */
+
++(void) registerForKeyboardType:(NSNumber *)type handler:(KeyboardAccessory)handler{
```

Comment:
Declare this method in the header so you don't need to do the NSSelectorFromString hack.

The API here should use `UIKeyboardType`. Keep NSNumber as an implementation detail.